CMakeLists.txt

```diff
@@ -34,6 +34,26 @@ SET(QMCPACK_VERSION_MINOR 7)
 SET(QMCPACK_VERSION_PATCH 9)
 SET(QMCPACK_VERSION "${QMCPACK_VERSION_MAJOR}.${QMCPACK_VERSION_MINOR}.${QMCPACK_VERSION_PATCH}")
 
+
```

Comment:
Could you move this to cmake/run_qe.cmake

Reply:
Done. ( The call is early in the cmake so that the user does not have to endure extensive cmake configuration before any abort if qe is not found. )

Reply:
However now, pw.x and and pw2qmcpack will be checked twice. I still don't like this place. If you think QE check needs to be a head, we can also have python check ahead. Let me think a bit more.
Currently run_qe is only used in pw2qmcpack tests. For this reason, it is natural to put it there.

Reply:
Why will pw.x be checked twice? I added a function that is called once as far as I can tell.

Reply:
run_qe is now our home for qe related macros, functionality.

Reply:
You are right. A function is good.